nixjs-rt/src/builtins.ts

```diff
@@ -354,12 +358,26 @@ export function getBuiltins() {
         throw builtinBasicTypeMismatchError("import", pathStrict, expected);
       }
 
-      const pathValue = pathStrict.toJs();
+      let pathValue = "";
+      if (pathStrict instanceof NixString) {
+        pathValue = normalizePath(pathStrict.toJs());
+      } else if (pathStrict instanceof Path) {
+        pathValue = pathStrict.toJs();
+      }
```

Comment:
Nitpick: you move the type exception from line 358 to here and remove the `if` block above:
```suggestion
      } else {
        const expected = [Path, NixString];
        throw builtinBasicTypeMismatchError("import", pathStrict, expected);
      }
```